### ISSUE TITLE
you require >= php 5.4 in your composer.json but you use the coalesce operator in src/Bitap/search.php

### ISSUE THREAD

**Issue by fbiesse** [Loilo/Fuse#3]
Good job, but I would like to use this library in php 5.6 but it is impossible because of the use of the coalesce operator that appears in php 7.
Do you think it is possible to make it compatible like you describe it in your composer.json ?

Thank you.

**Comment by Loilo** [comment]
Aw, I mentioned it in the readme but apparently forgot to update the composer.json.

Let's see what we can do.

**Comment by fbiesse** [comment]
Thank you, at the moment, I moved to version 2.6.2.

**Comment by Loilo** [comment]
Okay, I took a look.

First, I have to correct you: The composer.json correctly states that PHP 7 is required. The only version where this is broken is the first release of version 3, namely `3.0.5`, which was a bug and was immediately fixed in `3.0.6`.

Second, it took a while, but I made the tests pass in PHP 5.6 (wasn't easy, the [changed sorting behaviour](http://php.net/manual/en/migration70.incompatible.php#migration70.incompatible.other.sort-order) was trolling me a bit).

Long story short: Version `3.2.1` is released with re-introduced support for PHP 5.6. Have fun with it. 🙂

**Comment by fbiesse** [comment]
Thank you ! :)

**Comment by Loilo** [comment]
My pleasure. Would love to hear if everthing works as expected.
